packages/lesmis/src/components/DashboardExportModal/context/ExportOptionsContext.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
```

Comment:
Good idea to use context for the export options since the controls and the chart are so nested!